doc/guides/onboarding-extras.md

```diff
@@ -21,6 +21,8 @@ request.
 * `feature request` - Any issue that requests a new feature (usually not PRs)
 * `good first issue` - Issues suitable for newcomers to process
 * `meta` - For issues whose topic is governance, policies, procedures, etc.
+* `tsc-agenda` - Open issues and pull requests with this label will be added to
+  the Technical Steering Committee meeting agenda
```

Comment:
Not sure if it is covered in detail elsewhere in the onboarding docs... but if not, it would be good to highlight that `tsc-agenda` should be used sparingly and only if the issue cannot be resolved through normal discourse.